src/main/java/seedu/address/ui/DayCard.java

```diff
@@ -29,20 +33,25 @@
      * @see <a href="https://github.com/se-edu/addressbook-level4/issues/336">The issue on AddressBook level 4</a>
      */
 
-    private final DayOfWeek day;
+    private final int dayCardNumber;
 
     @FXML
     private VBox slotPane;
     @FXML
-    private Label dayLabel;
+    private Label dayDateLabel;
 
     /**
      * Creates a {@code PersonCode} with the given {@code Person} and index to display.
      */
-    public DayCard(DayOfWeek day, ObservableList<Person> stafflist, Period currentPeriod) {
+    public DayCard(int dayCardNumber, ObservableList<Person> stafflist, Period currentPeriod) {
         super(FXML);
-        this.day = day;
-        dayLabel.setText(day.toString().substring(0, 3));
+        this.dayCardNumber = dayCardNumber;
+        DayOfWeek day = LocalDate.now().getDayOfWeek().plus(dayCardNumber);
```

Comment:
Should use the period input to change as the we have a command to change which week to look at.
I think the period::toList() function would help. 
Maybe retain the DayOfWeek input to compare with period::toList?

Reply:
Aighto, forgot about the Change command, have just updated it, if you wanna take a look.